mods/d2k/chrome/ingame-player.yaml

```diff
@@ -271,9 +271,24 @@ Container@PLAYER_WIDGETS:
 									X: 5
 									Y: 5
 									ImageCollection: production-icons
+						ProductionTypeButton@UPGRADE:
+							Key: e
```

Comment:
Please also use a configurable HotkeyName like we did in https://github.com/OpenRA/OpenRA/pull/7541.